tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/MoneyLaunderingOperation.java

```diff
@@ -223,7 +223,9 @@ public void close(HeisenbergConnection heisenbergConnection) throws MuleExceptio
     };
   }
 
-  public PagingProvider<HeisenbergConnection, Integer> failAtClosePagedOperation(Integer failOn) {
+  public PagingProvider<HeisenbergConnection, Integer> failingConnectivityPagedOperation(Integer failOn,
```

Comment:
By changing the old operation, you had to change all the tests that used it, why not create a new one?

Reply:
For reuse because the new operation I needed was almost identical to this one and I was trying to reduce the bloat on the heisenberg connector, and it was easy to change the tests that used it.

Reply:
revert this change. You're adding tests at the expense of others. These are not the same scenario